Parsers/ASimNetworkSession/ProductParsers/vimNetworkSessionMicrosoft365Defender.yaml

```diff
@@ -27,7 +62,25 @@ ParserQuery: |
   ];
   // -- Common preprocessing to both input and outbound events
   let RawNetworkEvents =
-    DeviceNetworkEvents
+    DeviceNetworkEvents | where not(disabled)
+    // ***************  Prefilterring *****************************************************************
+    |where (isnull(starttime) or TimeGenerated>=starttime) and
+           (isnull(endtime) or TimeGenerated<=endtime) and
```

Comment:
Until now I think we put the logical operators at the beginning of the following line. Not critical, but I would like to keep consistency.